docs/content/en/docs/implementing/day-2-operations/_index.md

```diff
@@ -0,0 +1,295 @@
+    metadata:
+      labels:
+        app.kubernetes.io/name: podtato-head-hat
+        app.kubernetes.io/part-of: podtato-head
+        app.kubernetes.io/version: 0.1.1
+    spec:
+      containers:
+        - name: podtato-head-hat
+          image: podtato-head-hat:a
+```
+
+Now, let's assume that the configuration of that workload needs to be changed.
+In this example we assume that the image of that workload
+should be updated, but a configuration change is not limited to that.
+From here, you essentially have two options:
+
+* **Only update the configuration *without* updating the `app.kubernetes.io/version`
+label:** This can be useful when the change in the configuration should happen regardless
+of the result of any task or evaluation, e.g., when the previously used image has a critical vulnerability
+and the image must be updated as quickly as possible.
+To do that, change `podtato-head-frontend` as follows:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: podtato-head-frontend
+  namespace: podtato-kubectl
+spec:
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: podtato-head-frontend
+        app.kubernetes.io/part-of: podtato-head
+        app.kubernetes.io/version: 0.1.0 # The version label stays the same and therefore no related KeptnWorkload will be affected
+        keptn.sh/pre-deployment-evaluations: my-evaluation
+        keptn.sh/pre-deployment-tasks: my-task
+    spec:
+      containers:
+        - name: podtato-head-frontend
+          image: podtato-head-frontend:b # Only the image tag has been updated from 'a' to 'b'
+```
+
+* **Update the configuration *and* the version label:**
+   Doing so causes the`KeptnWorkload` that is associated
+    with the `podtato-head-frontend` deployment to be updated,
+    and therefore the pre-task `my-task` and pre-evaluation `my-evaluation`
+    are executed before the updated pods are scheduled.
+In this case, the deployment should be changed as follows:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: podtato-head-frontend
+  namespace: podtato-kubectl
+spec:
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: podtato-head-frontend
+        app.kubernetes.io/part-of: podtato-head
+        app.kubernetes.io/version: 0.2.0 # The version label is now updated as well
+        keptn.sh/pre-deployment-evaluations: my-evaluation
+        keptn.sh/pre-deployment-tasks: my-task
+    spec:
+      containers:
+        - name: podtato-head-frontend
+          image: podtato-head-frontend:b # The image tag has been updated from 'a' to 'b'
+```
+
+If you have defined the related `KeptnApp` resource yourself,
+this must also be updated to refer to the updated `KeptnWorkload`.
+This is a mandatory step, since the `KeptnWorkload` associated with
+this updated deployment is not able to progress otherwise.
+Therefore, make sure that the version of `podtato-head-frontend`
+is updated accordingly:
+
+```yaml
```

Comment:
please use external asset files and include them here for all the code examples in this file :)